### ISSUE TITLE
Make mightyiam an owner of this repository

### ISSUE THREAD

**Issue by mightyiam** [standard/eslint-config-standard-with-typescript#599]
For #598, please make me an owner of this repository.

@standard/team

**Comment by mightyiam** [comment]
@bcomnes @feross @Flet @LinusU @mafintosh @ungoldman @watson @yoshuawuyts forgive the group ping. Someone make me an owner of this repository, please.

**Comment by mightyiam** [comment]
@bcomnes @feross @Flet @LinusU @mafintosh @ungoldman @watson @yoshuawuyts if you read this, make me an owner of this repository, please.

**Comment by voxpelli** [comment]
@mightyiam Can you describe _why_ you need to be made an owner? That will likely increase the likelihood of that happening.

My understanding from #598 is that it is to set up TravisCI.com, but eg. both me and @Divlo have responded there about the possibilities of using GH Action instead, but without a response from you?

GH Action:s would be possible without an added ownership and thus without any dependencies on the action of another maintainer + would be more in line with the rest of the repositories in the `standard` organization, which would be a good bonus as far as I'm concerned.

**Comment by bcomnes** [comment]
@mightyiam  Apologies, I have 0 context around this package.  It appears you are the original author of this, correct? I've made you the maintainer, which should give you everything you need to do to change integrations, change settings, etc short of deleting the package.

**Comment by mightyiam** [comment]
@voxpelli 

> @mightyiam Can you describe _why_ you need to be made an owner? That will likely increase the likelihood of that happening.

For re-enabling Travis CI to solve #598.

> My understanding from #598 is that it is to set up TravisCI.com, but eg. both me and @Divlo have responded there about the possibilities of using GH Action instead, but without a response from you?

> GH Action:s would be possible without an added ownership and thus without any dependencies on the action of another maintainer + would be more in line with the rest of the repositories in the `standard` organization, which would be a good bonus as far as I'm concerned.

I reviewed the PR to migrate to GitHub actions and rejected it for [this reason](https://github.com/standard/eslint-config-standard-with-typescript/pull/530#discussion_r582633766).

@bcomnes 

> @mightyiam Apologies, I have 0 context around this package. It appears you are the original author of this, correct? I've made you the maintainer, which should give you everything you need to do to change integrations, change settings, etc short of deleting the package. Ill see if I can track down feross and check if its okay to go up one more level to repo admin if you want.

Thanks. Yes, I am the original author. Apparently, whatever permission you gave me doesn't allow me to re-enable Travis CI. Only an owner is allowed:

https://travis-ci.com/github/standard/eslint-config-standard-with-typescript

![Screenshot from 2021-06-30 09-20-58](https://user-images.githubusercontent.com/635591/123892298-81b58700-d984-11eb-9ea8-34d4775789b6.png)

---

I don't understand the hesitation. Development is blocked. I'm not trying to sabotage this project. Please, the first person who is able to, go in the settings of this repository and make me an owner.

**Comment by bcomnes** [comment]
Ok try now, you are full admin on the repo. Apologies again Github makes it unfortunately unclear what the different levels actually allow, and I'm not typically involved in making these determinations. It is your package though and you should have full access, 100% agreed. I'm just trying to do due diligence as we haven't crossed paths before. (Good to meet btw!)

**Comment by mightyiam** [comment]
@bcomnes thank you. Yet, still seems insufficient.

Apparently, only an organization owner may perform the necessary operation. Please either make me an organization owner or try to perform the necessary actions yourself or anyone else reading this:

While I'm a long time user of Travis CI, I didn't take the time to understand their various old/new repo access paradigms. From intuition, here's what should work:
1. Install the GitHub app here:
https://github.com/apps/travis-ci/installations/8469464
1. Migrate this repository here:
https://travis-ci.com/organizations/standard/migrate

**Comment by voxpelli** [comment]
> I don't understand the hesitation. Development is blocked. I'm not trying to sabotage this project. Please, the first person who is able to, go in the settings of this repository and make me an owner.

I just want to clarify that I don't have high enough permissions in this organization to give any level of permissions to anyone for anything, I simply have write permissions, hence why I tried to help unblock development in other ways.

Eg. describing the urgency + why it should be a no-brainer (eg. that it's your project to begin with) helps a lot with people unfamiliar with the repo in understanding and taking action.

Generally, keeping ownership level permission tight is a good security strategy for any project, so some hesitation is good.

---

Maybe raise a new issue about adding the TravisCI app or rename this one? As else the notifications will not highlight the shifted needs.

**Comment by bcomnes** [comment]
I seem to be able to approve the request, and I have. Let me know if it works or if you need more perms.

**Comment by mightyiam** [comment]
@bcomnes that's progress. Now a migration from the old infrastructure to the new seems to be necessary. According to what you did so far, you should have the permissions to do this, as well:
https://travis-ci.com/organizations/standard/migrate

**Comment by bcomnes** [comment]
I will try today.

**Comment by bcomnes** [comment]
Ok, I migrated it to the travis beta thing.  Did that get it to work?

**Comment by mightyiam** [comment]
@bcomnes yes. Thank you and everyone.
